plugin_info.toml

```diff
@@ -13,8 +13,10 @@ license = 'MIT'
 
 [plugin-install]
 #packages required for your plugin:
+packages-required = ['pymodaq>=4.0.0', 'daqhats']
 packages-required = ['pymodaq>=4.0.0', 'picamera2']
```

Comment:
there should be only one list including all necessary packages, here you created two similar lines...